sdk/js/package.json

```diff
@@ -63,7 +63,7 @@
   "dependencies": {
     "@certusone/wormhole-sdk-proto-web": "^0.0.5",
     "@certusone/wormhole-sdk-wasm": "^0.0.1",
-    "@injectivelabs/sdk-ts": "^1.0.75",
+    "@injectivelabs/sdk-ts": "^1.0.211",
```

Comment:
Do we want to make sure that the client is also on this set of injective releases?  Should that be a clickup task?

Reply:
When the wormhole-sdk package is published to npm and the client updates to that version, we'll need to make sure the injective packages match